src/Ombi.Notifications/BaseNotification.cs

```diff
@@ -210,6 +214,13 @@ protected UserNotificationPreferences GetUserPreference(string userId, Notificat
                 .FirstOrDefault(x => x.Agent == agent && x.UserId == userId);
         }
 
+        protected async Task<IEnumerable<OmbiUser>> GetPowerUsers()
```

Comment:
This is not really getting power users, It's getting all users that have raised privilidges